catalog/ui/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceConstants.java

```diff
@@ -18,11 +18,11 @@
 
   public static final String WORKSPACE_METACARDS = "metacards";
 
-  public static final String WORKSPACE_QUERIES = "queries";
```

Comment:
❓ Do we need to leave the queries field on the metacard for backwards compatibility?

Reply:
In a meeting that we had about a week ago we decided to go forward with data migration instead of backwards compatibility

Reply:
For the curious, what was the reasoning behind that decision?

Reply:
I'm going to summarize what @millerw8 has spoke to. The main concern was from a reliability and maintainability standpoint. If we support backwards compatibility how long will that happen and when can we safely remove the old format? Migration is the lower impact change compared to supporting two formats for query metacards.